staging/mssqldb.tf

```diff
@@ -7,21 +7,8 @@ resource "aws_db_subnet_group" "mssql_db_subnets" {
   }
 }
 
-# identifier for the Source database
-data "aws_db_instance" "source_db" {
-  db_instance_identifier = "${var.mssql-db-source}-${var.environment_name}"
-}
-
-# Snapshot1 - create a snapshot of the Source DB
-resource "aws_db_snapshot" "db1_snapshot" {
-  db_instance_identifier = data.aws_db_instance.source_db.id
-  db_snapshot_identifier = "${var.mssql-db-target}-snapshot"
-}
-
 # use Snapshot1 to create a database with EE instance
 resource "aws_db_instance" "hfs-mssql-web" {
-  snapshot_identifier = aws_db_snapshot.db1_snapshot.id
-
```

Comment:
PR description doesn't mention these changes:

![image](https://github.com/LBHackney-IT/mtfh-finance-infrastructure/assets/43747286/36af843a-28d3-41f0-8a5e-2108240c4a37)

What is this about?

Reply:
Staging was failing to Terraform Plan because we've deleted the database that we created a snapshot for.

I looked online and they suggest just detaching and deleting the snapshot and the data will persist in the new DB